src/ros.h

```diff
@@ -320,7 +321,7 @@ class RosSignal : public QQuickItem {
 
  Q_OBJECT
     Q_PROPERTY(QString topic WRITE setTopic MEMBER _topic)
-
+    Q_PROPERTY(bool received MEMBER _received NOTIFY onReceivedChanged)
```

Comment:
What about calling the notification function `onTriggered` instead? Might be a bit more readable